### PR TITLE
[Docs]: new README_unix.md to be included into tar or zip archives, c…

### LINKED ISSUE #8160: Binary release tarballs should come with a README
A typical release tarball (or zipball) currently contains the following:

```
bitcoin-0.12.1/
bitcoin-0.12.1/bin/
bitcoin-0.12.1/bin/bitcoin-cli
bitcoin-0.12.1/bin/bitcoind
bitcoin-0.12.1/bin/bitcoin-qt
bitcoin-0.12.1/bin/bitcoin-tx
bitcoin-0.12.1/bin/test_bitcoin
bitcoin-0.12.1/include/
bitcoin-0.12.1/include/bitcoinconsensus.h
bitcoin-0.12.1/lib/
bitcoin-0.12.1/lib/libbitcoinconsensus.so
bitcoin-0.12.1/lib/libbitcoinconsensus.so.0
bitcoin-0.12.1/lib/libbitcoinconsensus.so.0.0.0
```

I think we should add a README.md with the release that describes how to get started, and what every file is for.

For windows we have `doc/README_windows.txt` which gets packaged as part of the installer. It is not packaged as part of the windows zips though.

### LINKED ISSUE #8161: Binary release doesn't make it easy to find the source code
This has been suggested before, but I cannot find an issue for it.

I think we should include a link to this repository in various places. At least:
- [x] The "About" dialog in the GUI [done in #8207]
- [ ] A README that comes with release tarballs (see #8160)

This makes it easier to find the source code for potential contributors, as well as avoids awkward questions like #8157.

### PR DESCRIPTION
**Pull request by cisba**
…ontaining link to sources repo, basic info, instructions to run the executable files, files list and description, licence.

    This fixes #8160, fixes #8161, fixes #11971

### CODE REVIEW

File: doc/README_unix.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+Running
+---------------------
+
+Unpack the files into a directory and run:
+
+- `bin/bitcoin-qt` (GUI) or
+- `bin/bitcoind` (headless)
+
+Note: bitcoin-qt is missing in arm tars.
```
Comment (laanwj) [inline doc/README_unix.md]: I think this was resolved, at least for ARM64, in #13696

File: doc/README_unix.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+### Files
+```
+./bin/bitcoin-cli (Command Line Interface)
```
Comment (laanwj) [inline doc/README_unix.md]: would prefer this formatted as a list

- `./bin/bitcoin-cli` (Command Line Interface)
- `./bin/bitcoind` (headless daemon)
...

**Comment by laanwj** [comment]
not entirely decided on whether we want more platform-dependent READMEs: e.g. would be better to have one that is actively updated for every platform than heaps of them that eventually run out of date with teach other.

I think the main README.md already is, effectively, the UNIX one! so might be better to update that one.

The only reason that windows has a special README is because it deviates from the way things are done on UNIX-ish OSes in a few places.

**Comment by cisba** [comment]
Ok, I will make a new PR updating the main README.md

**Pull request closed without merging**